propagator/jaeger/lib/opentelemetry/propagator/jaeger/operation.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# OpenTelemetry is an open source observability framework, providing a
+# general-purpose API, SDK, and related tools required for the instrumentation
+# of cloud-native software, frameworks, and libraries.
+#
+# The OpenTelemetry module provides global accessors for telemetry objects.
+# See the documentation for the `opentelemetry-api` gem for details.
+module OpenTelemetry
+  # Namespace for OpenTelemetry propagator extension libraries
+  module Propagator
+    # Namespace for OpenTelemetry Jaeger propagation
+    module Jaeger
+      # Common superclass for classes that perform propagator operations, i.e.
+      # TextMapExtractor and TextMapInjector
+      class Operation
```

Comment:
There's really no reason for this class. You can put these constants on the Jaeger module or in the injector / extractor depending on whether they need to be shared in both places.